packages/@angular/cli/blueprints/ng2/files/__path__/tsconfig.app.json

```diff
@@ -0,0 +1,28 @@
+{<% if (ng4) { %>
+  "extends": "<%= relativeRootPath %>/tsconfig.json",
+  "compilerOptions": {
+  "lib": [
+    "es2016",
+    "dom"
+  ],<% } else { %>
+  "compilerOptions": {
+    "sourceMap": true,
+    "declaration": false,
+    "moduleResolution": "node",
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "lib": [
+      "es2016",
+      "dom"
+    ],<% } %>
+    "outDir": "<%= relativeRootPath %>/out-tsc/app",
+    "target": "es5",
+    "module": "es2015",
+    "baseUrl": "",
+    "types": []
```

Comment:
A couple of reasons:
- Some typings might actually have duplicate indentifiers. This has happened before with `protractor` and `jquery` both using `$`.
- You might be using typings that really don't shouldn't in your app, like `describe` that should only be in unit tests, or node typings.

That being said, it is more convenient to use `typeRoots` if you're not too concerned with those things.